lib/ansible/cli/doc.py

```diff
@@ -89,8 +88,8 @@ def _load_argspec(self, role_name, collection_path=None, role_path=None):
         else:
             raise AnsibleError("A path is required to load argument specs for role '%s'" % role_name)
 
-        loader = DataLoader()
-        return loader.load_from_file(path, cache=False, unsafe=True)
+        with open(path, 'r') as f:
+            return yaml.safe_load(f)
```

Comment:
use `from_yaml` function

Reply:
another reason for dataloader is path magic, but that is not needed here.

Reply:
👍

Reply:
dont use yaml direclty, use `from_yaml` as it will handle prioritizing C libyaml and other things (like always being safe)